docs/content/packages/oauth.md

```diff
@@ -248,6 +248,7 @@ app.get('/slack/oauth_redirect', (req, res) => {
 });
 ```
 </details>
+
 ---
```

Comment:
Fixes possible confusion in markdown parsing - this might've been interpreted as a header called "</details>"!